components/header-bar/src/header-bar.js

```diff
@@ -113,9 +106,7 @@ export const HeaderBar = ({
                     )}
                 </div>
 
-                {(pwaEnabled || showOnlineStatus) && !loading && !error && (
-                    <OnlineStatus dense />
-                )}
+                {pwaEnabled && !loading && !error && <OnlineStatus dense />}
```

Comment:
I don't understand what the point of this is 🤔? The logical condition seems to be the same as above and the `dense` property means that the css class used then includes `display:none`?

Reply:
It's part of making the `<OnlineStatus>` component reusable for the design in two different screen sizes:

<img width="422" alt="Screenshot 2023-03-17 at 1 49 03 PM" src="https://user-images.githubusercontent.com/49666798/225909726-c216514f-90fc-42f9-919f-ba156ebbe69e.png">
<img width="692" alt="Screenshot 2023-03-17 at 1 49 12 PM" src="https://user-images.githubusercontent.com/49666798/225909752-90de3d7a-2945-474a-8369-c56a51663230.png">

The first `<OnlineStatus>` is the "badge" and shows on larger screens and is hidden on larger screens; the second `<OnlineStatus>` (with the `dense` prop) is a "bar" that's hidden on larger screens and shown on smaller screens: [effect of `dense` prop on container](https://github.com/dhis2/ui/blob/LIBS-315-connection-status-in-headerbar/components/header-bar/src/online-status.js#L20), and [CSS with media query](https://github.com/dhis2/ui/blob/7887cfa019e9b2cab71b955fea987c4f81980b0f/components/header-bar/src/online-status.styles.js#L14-L36)

Reply:
Cool. Thanks. Missed the @media rule when reviewing 😅